lib/cmd/fh3/appforms/forms/export.js

```diff
@@ -0,0 +1,35 @@
+var fs = require('fs');
+var fhreq = require('../../../../utils/request.js');
+
+module.exports = {
+  'desc': 'Export all the forms defined in a given domain.',
+  'examples': [{
+    cmd: 'fhc appforms forms export --file=<Output zip file>',
+    desc: 'Export all the forms contained in your domain to an output zip file.'
+  }],
+  'demand': ['file'],
+  'alias': {},
+  'describe': {
+    'file': 'Output zip file containing the forms export.'
+  },
+  'url': '/api/v2/appforms/forms/export',
+  'method': 'get',
+  'customCmd': function (params, cb) {
+
+    params.url = this.url;
+    params.method = this.method;
+    params.output = params.file;
+
+    if (params.file.indexOf(".zip") === -1) {
```

Comment:
This is fragile, there's path.extname(). I think this fails for "some/my.zip/directory/", for instance